mv/MBS_MobileDirPad.js

```diff
@@ -221,64 +567,156 @@ MBS.MobileDirPad = {};
 	//
 	// Sprite for the action button
 
-	function Sprite_Button() {
+	function Sprite_mbsButton() {
 		this.initialize.apply(this, arguments);
 	}
 
-	Sprite_Button.prototype = Object.create(Sprite_Base.prototype);
-	Sprite_Button.prototype.constructor = Sprite_Button;
+	Sprite_mbsButton.prototype = Object.create(Sprite_Base.prototype);
+	Sprite_mbsButton.prototype.constructor = Sprite_mbsButton;
 
-	Sprite_Button.prototype.initialize = function(type) {
+	Sprite_mbsButton.prototype.initialize = function(type) {
 		Sprite_Base.prototype.initialize.call(this);
 		this._type = type;
-		if ((type == 0 ? $.Param.button : $.Param.cButton) == "") this.visible = false;
-		else
-			this.bitmap = ImageManager.loadNormalBitmap(type == 0 ? $.Param.button : $.Param.cButton, 0);
+		
+		let imgPath = '';  // default, empty string
+	switch (type) {
```

Comment:
This switch statement here is unncessary and better expressed using an Array, I think.
Instead of this huge clause, you could've just create an array for all images and used type as an index:
```javascript
let p = MBS.MobileDirPad.Param;
MBS.MobileDirPad.buttonImages = [p.actionButtonImage, p.cancelButtonImage, ...]
```
(sample code)

Better yet, you could have `type` be an actual identifier like `action`, `cancel`, etc. and use this to dynamically choose the appropriate image. Something like `MBS.MobileDirPad.Param[type + "ButtonImage"]`. Just make sure you check `type` before to avoid bad indices.